volcli/commands.go

```diff
@@ -34,6 +34,7 @@ var Commands = []cli.Command{
 		Subcommands: []cli.Command{
 			{
 				Name:        "upload",
+				Flags:       VolmasterFlags,
```

Comment:
I've moved this to the GlobalFlags, but we'll still be forced to specify GlobalFlags for every sub-command. This change will be in the next code push.